common/laws/00_rights_of_women.txt

```diff
@@ -247,7 +248,7 @@ law_womens_suffrage = {
 	}
 	
 	modifier = {
-		country_bureaucracy_investment_cost_factor_mult = 0.05
+		country_bureaucracy_investment_cost_factor_mult = 0.15
```

Comment:
why are we doing this? Like how do we want to influence player decision by doing this?

Reply:
Create more drawbacks for having a really powerful law. The 15% extra workforce and political participation (usually resulting in stronger trade unions and such) leads to other progressive laws. Additionally, it reflects the historical cost of having more workers requiring more bureaucracy to accommodate them.

That being said, is there soemthing about it that stands out to you as being "too much" or problematic compared to the original 5% boost?